package.json

```diff
@@ -222,8 +198,5 @@
   "engines": {
     "npm": ">=2.0.0"
   },
-  "typings": "./dist/package/Rx.d.ts",
```

Comment:
If `rxjs/Rx` is no longer a valid import, is `./dist/package/Rx.d.ts` going to be were the types are?

Reply:
Hmmm... that's a good question. I think we're going to need to take another pass through the module settings in package.json to ensure we're going about this right. We have have a little time, we just need to sort it out before the first alpha release.